ca_tools/scripts/control_node.py

```diff
@@ -44,142 +39,133 @@ class ctrl_Node:
     def __init__(self):
         
         rospy.init_node('control_node')
-        pubtopic = "/create1/cmd_vel"
-        self.my_pub = rospy.Publisher(pubtopic, Twist, queue_size=10)
-        subtopic = "/create1/gts"
+    
+        self._my_pub = rospy.Publisher(pubtopic, Twist, queue_size=10)
```

Comment:
Didn't you have any issue calling `pubtopic` like this?

Reply:
Adressed